Example/Utils/Bundle+MiniApp.swift

```diff
@@ -19,13 +19,31 @@ extension Bundle {
 
        // Get the URL to the resource bundle within the bundle
        // of the current class.
-       guard let resourceBundleURL = myBundle.url(
-               forResource: name, withExtension: "bundle")
-       else { fatalError("\(name).bundle not found!") }
+       var resourceBundleURL = myBundle.url(forResource: name, withExtension: "bundle")
+       if resourceBundleURL == nil {
+           resourceBundleURL = myBundle.resourceURL?.appendingPathComponent("Frameworks/MiniApp.framework/\(name).bundle")
+       }
+
+       guard let finalURL = resourceBundleURL
+       else {
+           print("🛑 \(name).bundle not found!")
+           #if SWIFT_PACKAGE
+           return Bundle.module
+           #else
+           fatalError("could not find resource bundle url")
+           #endif
+       }
 
        // Create a bundle object for the bundle found at that URL.
-       guard let resourceBundle = Bundle(url: resourceBundleURL)
-       else { fatalError("Cannot access \(name).bundle!") }
+       guard let resourceBundle = Bundle(url: finalURL)
+       else {
+           print("🛑 Cannot access \(name).bundle!")
+           #if SWIFT_PACKAGE
+           return Bundle.module
+           #else
+           fatalError("could not load resource bundle")
+           #endif
+       }
```

Comment:
here I just copy/pasted the code we're using in the SDK, the demo app was not up to date for SPM